fcrepo-exts/migration-utils
# Add id-prefix param

Add id-prefix param
* * *

**JIRA Ticket**: (https://jira.lyrasis.org/browse/FCREPO-3624)

# What does this Pull Request do?
Lets the user specify a different prefix to add to the fedora 3 pid for the ocfl object id.

# How should this be tested?
Tests are included, but to test manually, run a migration with --id-prefix "" and verify that the ocfl object id doesn't include "info:fedora/".

# Interested parties
@fcrepo/committers
